README.md

```diff
@@ -1,21 +1,23 @@
+# Ninja
```

Comment:
Don't know how to get the URL for AppVeyor

Reply:
https://www.appveyor.com/docs/status-badges/

But I don't know how to get the project ID part. 

This might require the person who set up the AppVeyor project, or an appropriately permissioned account, to log in and get it.

Reply:
Okay won't happen then any time soon. It isn't that important anyway, only nice to have.